demos/SanicJymper/main.cpp

```diff
@@ -210,26 +229,26 @@ void onWindowKeyboard(unicorn::system::Window::KeyboardEvent const& keyboardEven
             pWindow->SetMouseMode(MouseMode::Captured);
             break;
         }
-    case Key::V:
-    {
-        if (Action::Repeat == action)
+        case Key::V:
         {
-            return;
+            if(Action::Repeat == action)
+            {
+                return;
+            }
+            depthTest = !depthTest;
+            pGraphics->SetDepthTest(depthTest);
+            break;
         }
-        depthTest = !depthTest;
-        pGraphics->SetDepthTest(depthTest);
-        break;
-    }
-    case Key::Escape:
+        case Key::Escape:
         {
             pWindow->SetMouseMode(MouseMode::Normal);
             break;
         }
         case Key::Insert:
         {
-            if (pInput)
+            if(pInput)
```

Comment:
Please don't beautify the code you aren't modifying. It inflates commits and makes for more merge conflicts. Same applies to all other files.